tests/common/common.h

```diff
@@ -153,20 +153,24 @@ void check_get_profiling_info_param(sycl_cts::util::logger& log,
  * @brief Helper function to check the equality of two SYCL objects.
  */
 template <typename T>
-void check_equality(sycl_cts::util::logger& log, T& a, T& b, bool checkGet) {
+void check_equality(sycl_cts::util::logger& log, T& a, T& b) {
   /** check is_host
    */
   if (a.is_host() != b.is_host()) {
     FAIL(log, "two objects are not equal (is_host)");
   }
 
-  /** check get
+#ifdef SYCL_BACKEND_OPENCL
+  /** check get_native
    */
-  if (checkGet) {
-    if (a.get() != b.get()) {
-      FAIL(log, "two objects are not equal (get)");
+  auto queue = sycl_cts::util::get_cts_object::queue();
```

Comment:
I suggest moving the preprocessor check after the object, e.g.

```suggestion
  auto queue = sycl_cts::util::get_cts_object::queue();
  auto queueBackend = queue.get_backend();
#ifdef SYCL_BACKEND_OPENCL
```

The same elsewhere

Reply:
But if we move preprocessor check after the object we are going to get unused variables `queue` and `queueBackend` when SYCL_BACKEND_OPENCL is not defined, how code would benefit from such change?